config/import-control.xml

```diff
@@ -7,7 +7,21 @@
     <allow pkg="java.io"/>
     <allow pkg="java.util"/>
     <allow pkg="com.github.checkstyle.regression.data"/>
+    <allow pkg="com.github.checkstyle.regression.util"/>
+    <allow pkg="org.immutables"/>
+
     <subpackage name="git">
         <allow pkg="org.eclipse.jgit"/>
     </subpackage>
+
+    <subpackage name="module">
+        <allow pkg="com.google.gson"/>
+        <!-- Keep this until we don't need hardcode ModuleInfo map. -->
+        <allow class="java.nio.charset.Charset"/>
+    </subpackage>
+
+    <subpackage name="util">
+        <allow pkg="com.google.gson"/>
```

Comment:
Lets move all gson code to new component in #37 .
Either this component will now receive the collection of ModuleExtractInfos, or a utility class will and keep it as global information.